client/src/components/Site.vue

```diff
@@ -18,59 +18,59 @@
 
     <div>
       <p class="level-heading heading">Current Values</p>
-      <nav class="level">
+      <nav class="level is-mobile">
         <div class="level-item has-text-centered">
           <div>
             <p class="heading">Air Temp &deg;C</p>
-            <p class="title">{{ datums['Weather Air Temperature Value'].data | latestVal }}</p>
+            <p class="title">{{ this.$options.filters.latestResult(datums['Weather Air Temperature Value'].data, 'Value') }}</p>
           </div>
         </div>
         <div class="level-item has-text-centered">
           <div>
             <p class="heading">Pressure mbar</p>
-            <p class="title">{{ datums['Weather Barometric Pressure Value'].data | latestVal }}</p>
+            <p class="title">{{ this.$options.filters.latestResult(datums['Weather Barometric Pressure Value'].data, 'Value') }}</p>
```

Comment:
if this is a filter, it should be used as a filter, like this: `{{ datums['Weather Barometric Pressure Value'] | latestResult }}`

Reply:
Duly noted. VueJS shows that the two syntaxes are equivalent so I didn't think it mattered which was used; but its fixed in the latest commit.